packages/babel-core/src/config/loading/files/plugins.js

```diff
@@ -84,25 +79,69 @@ export function loadGenerator(name: string, dirname: string): { filepath: string
   };
 }
 
-function resolveQuiet(name: string, dirname: string): string|null {
-  try {
-    return resolve.sync(name, { basedir: dirname });
-  } catch (e) {
-    // The 'resolve' module can currently throw ENOTDIR
-    // https://github.com/substack/node-resolve/issues/121
-    if (e.code !== "MODULE_NOT_FOUND" && e.code !== "ENOTDIR") throw e;
-
-    // Silently fail and move to the next item.
-  }
-  return null;
+function standardizeName(type: "plugin"|"preset", name: string) {
```

Comment:
We do some similar plugin name normalization in `babel-preset-env` when handling includes/excludes (albeit much simpler).

Just thinking aloud, but it might be nice later to have this as some shareable util (especially if/when we move to `@babel/*